cache.js

```diff
@@ -1,14 +1,18 @@
 var path = require( 'path' );
 var crypto = require( 'crypto' );
 
+var defaultUseChecksum = !!process.env.FILE_ENTRY_CACHE_USE_CHECKSUM;
+
 module.exports = {
   createFromFile: function ( filePath, useChecksum ) {
+    if (defaultUseChecksum) useChecksum = true;
```

Comment:
may want to test that `useChecksum` is `undefined` before overwriting it? suppose it would be a matter of precedence between the env variable and this argument.. 

```suggestion
    if (defaultUseChecksum && typeof useChecksum === 'undefined') useChecksum = true;
```

Reply:
Personally, i think it should override whatever is passed. I was aiming for this env var to only ever matter if it is set - thus, removing the need to check for undefined.

Reply:
sure thing. i guess i was just thinking about backwards compatibility, but I suppose that would only be an issue in the cases where someone already has this env var set (for some reason) and is explicitly providing a value for `useChecksum`. in any event, we'd be happy with either implementation here.